{"cveId": "CVE-2020-12889", "cwe": [], "cvss": [{"baseScore": "9.8", "version": "3.1", "Attack Vector": "None", "Attack Complexity": "Low", "Privileges Required": "None", "User Interaction": "None", "Scope": "Unchaged", "Confidentiality Impact": "High", "Integrity Impact": "High", "Availability Impact": "High"}], "references": ["https://github.com/MISP/MISP-maltego/commit/3ccde66dab4096ab5663e69f352992cc73e1160b"], "description": ["MISP MISP-maltego 1.4.4 incorrectly shares a MISP connection across users in a remote-transform use case."], "published": "2020-05-15T18:15:13.727", "state": "PUBLIC", "vendorName": ["n/a"], "productName": ["n/a"], "github": {"advisories": [], "commits": ["https://github.com/MISP/MISP-maltego/commit/3ccde66dab4096ab5663e69f352992cc73e1160b"], "pocAdvisorie": null, "repo": "https://github.com/MISP/MISP-maltego/", "info": {"exist": true, "topics": ["analysis", "attack", "graph", "maltego", "misp", "misp-maltego", "mitre-attack", "pivoting", "threat-intel", "threat-intelligence", "transform", "visualisation"], "langs": {"Python": 67258, "Dockerfile": 1559, "Shell": 92}, "avatar": "https://avatars.githubusercontent.com/u/4134875?v=4", "stargazers": 174, "language": "Python"}}, "pocList": []}